autoinstall_script/AutoInstall.sh

```
@@ -0,0 +1,1150 @@
+
+STEP_BEGIN  "Checking configuration file"
+    if [[ ! -f "$CONFIG_FILE" ]]; then
+        STEP_FAIL  "Configuration file not found: $CONFIG_FILE. Ensure 'ivorysql.conf' exists in the script directory."
+    fi
+STEP_SUCCESS  "Configuration file detected"
+
+STEP_BEGIN  "Loading configuration file"
+if grep -Evq '^\s*([A-Z_][A-Z0-9_]*\s*=\s*.*|#|$)' "$CONFIG_FILE"; then
+    STEP_FAIL  "Invalid configuration format: only KEY=VALUE pairs and comments are allowed."
+fi
+source "$CONFIG_FILE" || STEP_FAIL  "Failed to load configuration file: $CONFIG_FILE. Check file format."
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)